src/pages/search.vue

```diff
@@ -156,9 +153,9 @@ export default defineComponent({
       }
       focusIn(document.getElementById('__layout'), Focus.First)
     },
-    fetchMedia(...args: unknown[]) {
+    fetchMedia({ shouldPersistMedia } = { shouldPersistMedia: false }) {
```

Comment:
Is `shouldPersistMedia` always `false` here or should the type be `boolean`?

```suggestion
    fetchMedia({ shouldPersistMedia } = { shouldPersistMedia: boolean }) {
```

Reply:
Oh dear, I totally misread this code! I thought it was the type but it was the default value :facepalm: Sorry for the bad suggestion :disappointed: